pkg/helm/vfs_renderer.go

```diff
@@ -145,3 +145,14 @@ func stripPrefix(path, prefix string) string {
 	pv := strings.Split(path, "/")
 	return strings.Join(pv[pl:], "/")
 }
+
+// list all the buildin profiles.
+func ListBuildinProfiles() []string {
```

Comment:
ProfileNames is map[string]bool for fast existence check, so values will always be true, you don't have to check for it.
You could make this a more general function (maybe in util) called StringBoolMapToSlice to make the code more reusable.

Reply:
done